### ISSUE TITLE
m3u plugin

### ISSUE THREAD

**Issue by bizzss** [cabernetwork/provider_video_m3u#4]
Hello, I have tried the current linux and windows version using m3u plugin and have the same problem.

Everything is installed correctly on both systems and I  generated a new  m3u file and it will not play in any player. Is this the correct generated format stream? 
(#EXTINF:-1 channelID='Foxnews1.us' tvg-num='10' tvg-chno='10' tvg-name='FOX NEWS (SD)' tvg-id='Foxnews1.us' tvg-logo='http://myiptv.com/logos/foxnewshdd.png' group-title='M3U',FOX NEWS (SD)
http://10.5.0.2:5004/M3U/watch/Foxnews1.us 

There are no errors reported in the logs.  If the stream is correct; what am I doing wrong?   
Thanks

**Comment by bizzss** [comment]
[cabernet.log](https://github.com/cabernetwork/provider_video_m3u/files/14173578/cabernet.log)
Thanks rocky for helping.  I tried every streaming setting with ffmpeg streamlink etc with both linux and windows. No streams will play on vlc or tvheadend.  Thought?

**Comment by bizzss** [comment]
Rocky, thanks for looking at that post again and seeing the attached debug.log. I couldnt understand what I did wrong and was going to ask you until I just seen your update. I used this program a few years ago with locast and it ran perfect. I cannot figure out what I am doing wrong.

Anyway,  attached is another debug file. I changed the streaming to ffmpeg and played one stream and still have the same results as before. All channels are showing in vlc but will not play. It just keeps on scrolling.  The m3u appears to be in the database because I can see them all within the edit screen within cabernet.

Thanks
[cabernet.log](https://github.com/cabernetwork/provider_video_m3u/files/14182164/cabernet.log)

**Comment by rocky4546** [comment]
ffmpeg has no logging in the log.  Just FYI, the cabernet.log will normally not have any log data from ffmpeg or streamlink.  That outputs to STDOUT and is only available from the System Log Handler and may be dependent on the log level setting.  The most log information is provided with the internal_proxy setting.  I would try that.

**Comment by bizzss** [comment]
I changed to internal_proxy setting and restarted as per your instructions.  log attached.  TY
[cabernet.log](https://github.com/cabernetwork/provider_video_m3u/files/14182632/cabernet.log)

**Comment by rocky4546** [comment]
It looks like your myiptv website is messed up.  The m3u8 request was http://myiptv:8080/live/myusername/mypass/109827.ts which does not look like a m3u8 file.  Next, your website is returning an empty response to the files listed in the m3u8 file from your website.  So it looks like its your myiptv website that is causing the issue.

**Comment by bizzss** [comment]
I guess I cannot use cabernet. Thank you for taking the time to help.

**Comment by bizzss** [comment]
After thinking about what you said about the provider, I edited me playlist and changed the format of the stream. My provider does not allow m3u downloads so I used wire shark to get the streams. I manually created my new playlist and modified the stream url to "live" and eliminated the .ts with replacing it with m3u8. Cabernet excepted my new m3u8 list and all streams played with in vlc. I found an easier method after I did all this work. Lol.

Thank you for your help!!
